src/components/DeviceSummaryTile.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import {
+  Card,
+  CardTitle,
+  CardBody,
+  Button,
+  Bullseye,
+  Spinner,
+  Stack,
+  StackItem,
+} from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+import { useSelector, shallowEqual } from 'react-redux';
+
+const DeviceSummaryTileBase = ({
+  orphaned,
+  active,
+  no_reports,
+  never_reported,
+}) => (
+  <Card>
+    <CardTitle>Device summary information</CardTitle>
+    <CardBody>
+      <Stack>
+        <StackItem>
+          <Button variant="link">{orphaned}</Button> orphaned devices
+        </StackItem>
+        <StackItem>
+          <Button variant="link">{active}</Button> active devices
+        </StackItem>
+        <StackItem>
+          <Button variant="link">{no_reports}</Button> devices has not reported
+          in the last 6 months
+        </StackItem>
+        <StackItem>
+          <Button variant="link">{never_reported}</Button> devices were
+          registered but never reported
+        </StackItem>
+      </Stack>
+    </CardBody>
+  </Card>
+);
+
+DeviceSummaryTileBase.propTypes = {
+  orphaned: PropTypes.number,
+  active: PropTypes.number,
+  no_reports: PropTypes.number,
+  never_reported: PropTypes.number,
+};
+
+const DeviceSummaryTile = () => {
+  const { isLoading } = useSelector(
+    ({ imagesReducer }) => ({
+      isLoading:
+        imagesReducer?.isLoading !== undefined
+          ? imagesReducer?.isLoading
+          : true,
+    }),
+    shallowEqual
+  );
+
+  if (isLoading) {
+    return (
+      <Card>
+        <CardTitle>Device summary information</CardTitle>
+        <CardBody>
+          <Bullseye>
+            <Spinner />
+          </Bullseye>
+        </CardBody>
+      </Card>
+    );
+  }
+  return (
+    <DeviceSummaryTileBase
+      orphaned={54}
+      active={130}
+      no_reports={3}
+      never_reported={31}
```

Comment:
We can use real data for these values, install `@redhat-cloud-services/host-inventory-client` and query for all devices registered with edge for the active section. For now we can add `N/A` for the rest of these values.